test/suite/test_hs33.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger, wttest
+from wtscenario import make_scenarios
+
+# test_hs33.py
+#    Check log disabled for the history store table in metadata.
+class test_hs33(wttest.WiredTigerTestCase):
+    name = "test_hs33"
+    logging = [
+        ('log-enabled', dict(logenabled=True)),
+        ('log-disabled', dict(logenabled=False)),
+    ]
+
+    scenarios = make_scenarios(logging)
+
+    # Setup custom connection config.
+    def conn_config(self):
+        if self.logenabled:
+            log_param = "log=(enabled=true)"
+        else:
+            log_param = "log=(enabled=false)"
+        return log_param
```

Comment:
I'm not sure this is what we want. I think we want `conn_config` to always having `log=(enabled=true)`. Otherwise changing the metadata is not meaningful.

I think that adding an entirely new test is not necessary. A test like `test_timestamp03.py` already uses timestamps and logged and not logged tables in all combinations. What if you just checked the metadata of the URIs in that test, including the HS?